internal/commands/root.go

```diff
@@ -268,6 +277,19 @@ func getFilters(cmd *cobra.Command) (map[string]string, error) {
 	return allFilters, nil
 }
 
+func validateExtraFilters(filterKeyVal []string) []string {
+	// Add support for state = exclude-not-exploitable, will replace all values of filter flag state to "TO_VERIFY;PROPOSED_NOT_EXPLOITABLE;CONFIRMED;URGENT"
+	if extraFilter[filterKeyVal[0]] != nil {
+		for privateFilter, value := range extraFilter[filterKeyVal[0]] {
+			if strings.Contains(filterKeyVal[1], privateFilter) {
+				logger.PrintfIfVerbose("Set filter from extra filters: [%s,%s]", filterKeyVal[0], value)
+				filterKeyVal[1] = strings.Replace(filterKeyVal[1], filterKeyVal[1], value, -1)
```

Comment:
```suggestion
				filterKeyVal[1] = strings.Replace(filterKeyVal[1], privateFilter, value, -1)
```

did you check a situation where there are more values with the 'exclude-not-exploitable', in this situation - all the value will be replaced, is it ok?

Reply:
Tamar, as we spoke, i will not do the changes.